packages/sync/src/links/LinksBuilder.ts

```diff
@@ -90,8 +90,6 @@ function createOfflineLink(config: DataSyncConfig) {
     listener: config.offlineQueueListener,
     networkStatus: config.networkStatus as NetworkStatus
   });
-  const localDirectiveFilterLink = new LocalDirectiveFilterLink();
-  let offlineLinks = ApolloLink.from([offlineLink, localDirectiveFilterLink]);
-  offlineLinks = ApolloLink.split((op: Operation) => isMutation(op) && !isOnlineOnly(op), offlineLinks);
```

Comment:
I kinda see what this stuff was doing. 
First it was making decision how queue was used (only for mutations and only when Operation is not online only).
We kinda missing that now.

Reply:
@wtrocki yes but in this scenario the mutation was still being added to the queue even when it was onlineOnly? the logic is off. 

now i do the check inside the offline link and don't enqueue. am i missing something or was that not the whole point of onlineOnly, that it was never queued and always just fired straight away?

Reply:
Yes. This was because split was implemented incorrectly: 
https://www.apollographql.com/docs/link/composition.html#directional